packages/worker/src/workflow/bundler.ts

```diff
@@ -102,6 +107,7 @@ export class WorkflowCodeBundler {
 
     // Cast because the type definitions are inaccurate
     return {
+      sourceMap: '',
```

Comment:
```suggestion
      sourceMap: 'this field is deprecated',
```

I'm envisioning devs logging this value, seeing it empty, and wondering what they did wrong.

Reply:
Good suggestion.